evm/contracts/examples/ConversionRate.sol

```diff
@@ -16,7 +16,7 @@ contract ConversionRate is ChainlinkClient, Ownable {
     uint256[] responses;
   }
 
-  uint256 public currentRate;
+  uint256 public currentAnswer;
```

Comment:
Notably, this contract [can't specify that it implements](https://github.com/ethereum/solidity/issues/3514) the CurrentAnswerInterface.